.sync.yml

```diff
@@ -6,7 +6,6 @@
     - set: debian10-64
```

Comment:
It looks like this entire file can be removed now.

Reply:
Removed this file in b898cf7bfe8b406b277bc313baf047a0076cfd04